odml/validation.py

```diff
@@ -363,14 +367,14 @@ def property_values_check(prop):
 
     if prop.dtype is not None and prop.dtype is not "":
```

Comment:
String comparisons shouldn't be done with `is` (or `is not`).

Reply:
As an additional comment: please always use pylint and fix any errors and warnings in your code changes before pushing to a PR.